onnxruntime/core/providers/cuda/atomic/common.cuh

```diff
@@ -1,5 +1,20 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
```

Comment:
Add the MIT License back.

Reply:
This format is copied from the inference kernel file... 
Non of them have MIT License after "/* Modifications Copyright (c) Microsoft. */"

Reply:
We should be consistent about this. Many of our source files do have MIT License after Copyright Microsoft.
but I think it's not necessary since we have the LICENSE file included in our repo.
+@pranavsharma for comments.

Reply:
This looks ok. This particular file is still covered by the original license and we're distributing it with modifications under MIT mentioned in the overall product LICENSE document.

Reply:
Thank you for the review.